src/module/data/actor-character.mjs

```diff
@@ -30,10 +28,23 @@ export default class GrimwildCharacter extends GrimwildActorBase {
 			})
 		});
 
-		schema.thorns = new fields.NumberField({
-			integer: true,
+		// should these have some sort of healing pool attached to them?
+		schema.bloodied = new fields.BooleanField();
+		schema.rattled = new fields.BooleanField();
```

Comment:
Yeah, I would adjust these so that they're pools since that's the most relevant thing to track on them. The logic would be similar to what we use for the challenge pools item type, so I would lean towards making an abstract PoolField type and use it in both places.